jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,559 @@
+		"id": "string"
+	},
+	"duration": 0,
+	"created_at": "2020-01-13T18:51:40Z",
+	"messages": [{
+		"type": "string",
+		"message": "string",
+		"reason": "string"
+	}],
+	"contexts": [{
+		"name": "string"
+	}],
+	"organization": {
+		"name": "string"
+	},
+	"queued_at": "2020-01-13T18:51:40Z",
+	"stopped_at": "2020-01-13T18:51:40Z"
+}
+```
+Notice in the example above that you will receive very specific information about your job, including specific project and workflow details for the job, the date and time the job started and then finished, and job-specific information such as the executor type used, current status of the job, and the duration of the job.
+
+For a more detailed breakdown of each value returned in this request, please refer to the [Get Job Details](https://circleci.com/docs/api/v2/#get-job-details) section of the *CircleCI API v2 Reference Guide*.
+
+
+## Download Artifacts
+
+The following section details the steps you need to follow to download artifacts generated when a job is run. We will go through the steps of returning a list of artifacts for a job, and then changing the command we run to download the full set of artifacts. If you are looking for instructions to download the latest artifacts for a pipeline, without needing to specify a job number, see our [API v1.1 guide](https://circleci.com/docs/2.0/artifacts/#downloading-all-artifacts-for-a-build-on-circleci) – keep checking back here as this functionality will be added to API v2 in the future.
+
+### Prerequisites
+
+Before making an API call, make sure you have met the following prerequisites:
+
+* You have set up a GitHub or BitBucket account with a repository to use with CircleCI.
+* You have completed CircleCI onboarding and you have a project setup.
+* You have a [personal API token](https://circleci.com/account/api).
+* You have been [authenticated](#get-authenticated) to make API calls to the server.
+
+### Steps
+
+1. First we will ensure your api token is set as an environment variable. You maybe have already done this during authentication, but if not, run the following command in your terminal, substituting your personal api token:
+
+    ```
+    export CIRCLECI_TOKEN={your_api_token}
+    ```
+
+2.  Next we need the job number for the job you want to get artifacts for. You can find job numbers in the UI - either in the breadcrumbs on the Job Details page, or in the URL.
+
+    INSERT SCREENSHOT HIGHLIGHTING JOB NUMBER
+
+3.  Next we can use `curl` to return a list of artifacts for a specific job. Note, you can format JSON responses by piping the `curl` command into the `jq` utility if you have it installed: `curl ... | jq`:
+
+    ```sh
+    curl -X GET https://circleci.com/api/v2/project/{VCS}/{org-name}/{repo-name}/{job-number}/artifacts \
+    -H 'Content-Type: application/json' \
+    -H 'Accept: application/json' \
+    -H "Circle-Token: $CIRCLECI_TOKEN" | jq\
+    ```
+
+    Check the table below for help with formatting your API call correctly.
```

Comment:
@rosieyohannan I really like how you have broken down the parameters here for the API call. Tyler and I did not do this for our API use cases, so do you think we should make this change as well? I am just thinking about consistency in the document.